### DISCUSSION TITLE
Changing the Title displayed on the meetings page

### DISCUSSION THREAD

**Discussion by DerekDen** [code4recovery/12-step-meeting-list#1119] [Q&A] [answered]
We are an English speaking intergroup in a Portuguese speaking country.  Despite our website being in English we still get questions from people "Is such and such meeting in English?".  I would like to change "Meetings" to "English Language Meetings".

Is the possible?

Thanks, Derek
<img width="757" alt="Screenshot 2023-04-21 at 11 18 58" src="https://user-images.githubusercontent.com/112857352/233612470-6a916bee-967d-42cd-bcec-36b5ab5a0dfa.png">

**Comment by kiyote33** [comment]
Hi @DerekDen,
One way you might achieve putting an "English Speaking Meetings" title on your meetings page is to add some custom HTML through the Meetings Top widget, like I've done in this image.
![image](https://user-images.githubusercontent.com/57125427/233650904-acb82375-3444-4dac-af63-3cc2f3d50441.png)
Hope this helps...

**Reply by DerekDen** [reply]
Thanks, that looks as if it would work for, but I'm stuck on what custom html to add.  I assume I need to set some variable rather than just draw the text.  Can you share the html you added.

**Comment by joshreisner** [comment, marked as answer]
Another option is you might link to this page instead as your "Meeting Guide Online" page: https://aaunityportugal.org/aa-meetings/?type=english

If you wanted it to read "English Language Meetings" exactly, you could customize the type by adding this code to your theme's functions.php:

```
if (function_exists('tsml_custom_types')) {
    tsml_custom_types(array(
        'EN' => 'English Language',
    ));
}
```

If you do that, your Meeting Guide Online URL would become `https://aaunityportugal.org/aa-meetings/?type=english-language`

**Reply by DerekDen** [reply]
Many thanks Josh, I got that working.
